tests/conftest.py

```diff
@@ -24,10 +24,13 @@
 else:
     HAS_PYARROW = True
 
+import fmu.dataio as dio
+from fmu.config import utilities as ut
 from fmu.dataio.dataio import ExportData, read_metadata
 
 logger = logging.getLogger(__name__)
 
+logger.setLevel("ERROR")
```

Comment:
I think we should avoid setting logging level. See also https://github.com/equinor/fmu-dataio/issues/270